modules/getting-started/pages/do-a-quick-install.adoc

```diff
@@ -7,10 +7,7 @@
 {description}
 Once you install Docker, you can use a single command to download and install a pre-configured version of Couchbase Server Enterprise Edition on your computer.
 
-NOTE: The Docker container image that is used in this topic is meant to set up a 'sandbox' version of Couchbase Server.
-It's based on the Enterprise Edition and contains scripts that automatically configure several Couchbase components during installation.
-Although this particular image works great for a demo, it is not meant for production.
-For information about using containers to run Couchbase Server in production, see the xref:cloud:couchbase-cloud-deployment.adoc[Cloud and Container Deployment Overview].
+
```

Comment:
I think this is actually a useful caveat. We need to emphasize that a Docker install is not suitable for production. But the link is hyper-specific and not useful. Reinstated the note but updated the link to point to the Install section.

```suggestion
NOTE: The Docker container image that is used in this topic is meant to set up a 'sandbox' version of Couchbase Server.
It's based on the Enterprise Edition and contains scripts that automatically configure several Couchbase components during installation.
Although this particular image works great for a demo, it is not meant for production.
For information about installing Couchbase Server in production, see xref:install:get-started.adoc[].
```